### PR TITLE
DDBEASY-32 - Waiting time when no items available.

### PR DESCRIPTION
**Pull request by guddo**
http://platform.dandigbib.org/issues/465

### CODE REVIEW

File: ding_availability.module [hunk 1]
```diff
@@ -206,6 +219,39 @@ function ding_availability_holdings($provider_ids) {
...
+    $days = ceil($days / 7);
+    $label = 'week';
+  }
+  else {
+    $label = 'day';
+  }
+
+  if ($days > 1) {
+    $label = $plurals[$label];
+  }
```
Comment (kasperg) [inline ding_availability.module:247]: Could we use [`format_interval()`](https://api.drupal.org/api/drupal/includes%21common.inc/function/format_interval/7) to achieve this?

File: templates/ding-holdings.tpl.php [hunk 2]
```diff
@@ -13,4 +13,5 @@
 $reserved_text = format_plural($reserved_count, 'There is 1 user in queue to loan the material.', 'There are @count users in queue to loan the material.');
 ?>
 <p><?php print "$total_text $reserved_text"; ?></p>
+<p><?php print $closest_loan; ?></p>
```
Comment (kasperg) [inline templates/ding-holdings.tpl.php:16]: I think we should ensure that `$closest_loan` is not empty before generating more DOM elements. The extra `<p>` tags may be empty if the provider does not return `available_from` information.

**Comment by kasperg** [comment]
@guddo Review complete. Looks great. I have two comments for you to take a look at.

**Comment by inlead** [comment]
PR moved to proper repository: https://github.com/ding2/ding2/pull/133.
@kasperg please review that PR and put your comments.

File: ding_availability.module (hunk 1 above)
Comment (inlead) [inline ding_availability.module:247]: https://github.com/easySuite/ding2/commit/2311533e8d9d052fe5c21127de0ec4dd830c5273

File: templates/ding-holdings.tpl.php (hunk 2 above)
Comment (inlead) [inline templates/ding-holdings.tpl.php:16]: https://github.com/easySuite/ding2/commit/fef90315d26e9b6b58036e1ac55399aedb7e3c90

**Pull request closed without merging**